### PR TITLE
fix: Dashboard default permission and signout fix

### PR DESCRIPTION
**Pull request by Riddhiagrawal001**
## Type of Change

- [X] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by JeevaRamu0104 — APPROVED** [review]

File: src/screens/UserManagement/PermissionUtils.res [hunk 1]
```diff
@@ -139,69 +139,85 @@ let getAccessValue = (~permissionValue: permissionType, ~permissionList) => {
...
-    fileRead: FileRead->getAccessValueFromPermission,
-    fileWrite: FileWrite->getAccessValueFromPermission,
-    analytics: Analytics->getAccessValueFromPermission,
-    threeDsDecisionManagerWrite: ThreeDsDecisionManagerWrite->getAccessValueFromPermission,
-    threeDsDecisionManagerRead: ThreeDsDecisionManagerRead->getAccessValueFromPermission,
-    surchargeDecisionManagerWrite: SurchargeDecisionManagerWrite->getAccessValueFromPermission,
-    surchargeDecisionManagerRead: SurchargeDecisionManagerRead->getAccessValueFromPermission,
-    usersRead: UsersRead->getAccessValueFromPermission,
-    usersWrite: UsersWrite->getAccessValueFromPermission,
+    paymentRead: getAccessValue(~permissionValue=PaymentRead, ~permissionList),
```
Comment (PritishBudhiraja) [inline src/screens/UserManagement/PermissionUtils.res:175]: Why this change needed?

Comment (Riddhiagrawal001) [reply src/screens/UserManagement/PermissionUtils.res:175]: There was a function inside a function and also we could have directly called the function .

Comment (PritishBudhiraja) [reply src/screens/UserManagement/PermissionUtils.res:175]: ~permissionList writing unnecessarily

Comment (Riddhiagrawal001) [reply src/screens/UserManagement/PermissionUtils.res:175]: Instead of calling the function for each permission individually, it could have been called once at the beginning of the function. This would reduce the need to iterate over the list of permissions multiple times.
